src/gleam/string.gleam

```diff
@@ -803,3 +803,17 @@ pub fn capitalize(s: String) -> String {
     _ -> ""
   }
 }
+
+pub fn from(anything) -> String {
+  do_from(anything)
```

Comment:
If you make `do_from` return a StringBuilder (so no calling `iolist_to_binary` in the Erlang) and then call `string_builder.to_string` here then it'll be faster.

Reply:
Great idea. I was able to remove a lot of casting to binary and slim the code down.